mindsdb/integrations/handlers/instatus_handler/tests/test_instatus_handler.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+from mindsdb.integrations.handlers.instatus_handler.instatus_handler import InstatusHandler
+from mindsdb.api.mysql.mysql_proxy.libs.constants.response_type import RESPONSE_TYPE
+import pandas as pd
+
+
+class InstatusHandlerTest(unittest.TestCase):
```

Comment:
Also, let's move this file to https://github.com/mindsdb/mindsdb/tree/staging/tests/handler_tests

Reply:
Done